ledger/src/blockstore.rs

```diff
@@ -7434,7 +7444,7 @@ pub mod tests {
     #[test]
     fn test_insert_multiple_is_last() {
         solana_logger::setup();
-        let (shreds, _) = make_slot_entries(0, 0, 20, /*merkle_variant:*/ true);
+        let (shreds, _) = make_slot_entries(0, 0, 19, /*merkle_variant:*/ true);
```

Comment:
any reason to change this from 20 -> 19?

Reply:
The test tries to insert a shred where `index >` already recorded `last_index`.
Because of the embedded chained Merkle root, shreds have smaller capacity, so with `20` this line generates the same number of shreds as L7460.
So we won't see any shred with `index > last_index` when inserting the second set of shreds.

So I reduced this to `19` so that the added assertion in L7461 still holds as before.